lib/simplifyExpression/basicsSearch/reduceMultiplicationByZero.js

```diff
@@ -1,25 +1,21 @@
+const {query: q} = require('math-nodes');
+const {rewriteNode} = require('math-rules');
+
 const ChangeTypes = require('../../ChangeTypes');
+const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
+// The math-rules rule for this didn't do the right thing when the '0' was
+// in the middle of the args list.
```

Comment:
I'd actually prefer to define as many rules in mathsteps as possible

You could remove this comment?